examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -122,9 +122,9 @@
      "text": [
```

Comment:
<p>1) Increase performance</p><p> </p><p>2) In variational algorithms, there <strong>are</strong> usually optimization</p><p> </p><p>3) ... Braket gate-based<strong><s>,</s></strong> superconducting ...</p><p> </p><p>4) replace this page link with a description of what the page is. We try to avoid saying (link here) or this link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>